src/client/components/FooterComponent/FooterComponent.js

```diff
@@ -14,19 +14,19 @@ export default function FooterComponent() {
         content="Home"
         imgSrc={homeIcon}
         alt="icon"
-        href="/home"
+        href="/"
       />
       <NavigationalButton
         content="Winners"
         imgSrc={cupIcon}
         alt="icon"
-        href="/leaderboard"
+        href="#"
       />
```

Comment:
Have you talked to Daniel about this links? I know we don't have a leaderboard or a profile page but maybe we should do ones.

Reply:
Yes, i have asked him he says to do so.